grpc.go

```diff
@@ -340,6 +359,10 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	if err := addDevices(req.Devices, req.OCI); err != nil {
```

Comment:
`addDevices` should be done **before** `addMounts` so that the latter can find the block device.

Reply:
Yes I have definitely missed that, and this is needed since we expect to be able to mount only after the devices have been discovered inside the VM.

Reply:
PR updated.